src/RGActionPatches.csproj

```diff
@@ -32,49 +32,49 @@
   </PropertyGroup>
   <ItemGroup>
     <Reference Include="Assembly-CSharp">
-      <HintPath>C:\Games\RoomGirl\BepInEx\unhollowed\Assembly-CSharp.dll</HintPath>
```

Comment:
can you move the reference changes out of this file, too? if you make a `RGActionPatches.csproj.user` you can put them there and they'll override the properties in this file. or you can remove this file from the PR and I'll manually add the compile tags after merging